kaggle-classification/keras-trainer/model.py

```diff
@@ -70,7 +77,7 @@ def train(self, train):
 
     callbacks = [
         ModelCheckpoint(
-            self.model_path, save_best_only=True, verbose=True),
+            TEMPORARY_MODEL_PATH, save_best_only=True, verbose=True),
```

Comment:
Please clean up these files once you're done with them to avoid having to manually delete them when running locally.